pkg/cmd/log.go

```diff
@@ -0,0 +1,50 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/ClusterHQ/dvol/pkg/api"
+	"github.com/spf13/cobra"
+)
+
+func NewCmdLog(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "log",
+		Short: "Show log",
+		Run: func(cmd *cobra.Command, args []string) {
+			if err := outputLog(cmd, args, out); err != nil {
+				fmt.Fprintln(os.Stderr, err.Error())
+				os.Exit(1)
+			}
+		},
+	}
+	return cmd
+}
+
+func outputLog(cmd *cobra.Command, args []string, out io.Writer) error {
```

Comment:
As @robhaswell pointed out in a previous branch cmd and args are not used to we can remove them from the signature